README.md

```diff
@@ -1 +1,182 @@
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio website] <a name="about-project"></a>
+
+**[Save the plannet conference-website is a project for a global conference on climate change and Carbon-Zero. It contains two pages i.e the home page and about me page.it is a responsive site for both mobile and desktop users.]**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">html</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">N/A</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Displays Home page]**
+  **[Displays About page]**
+  **[Mobile menu links to about page]**
+  **[Desktop home page links to about page]**
+  **[Diplays menu when Mobile hamburger is clicked]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+-This is my Capstone project [Live Demo Link](https://cosywasswa.github.io/Capstone-Project1/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:cosywasswa/Capstone-Project1.git
+```
+
+### To run the project
+
+run the index.html
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+Use index.html file.
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Cosmas wasswa**
+
+- GitHub: [@githubhandle](https://github.com/cosywasswa)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/cosmas-wasswa-931a2761/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Reservation page]**
+- [ ] **[Contact page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Acknowledgement goes to cindy shin for making this website template design available status
```

Comment:
- [ ] [OPTIONAL] I noticed that you forgot to add a link to Cindy Shin’s original designs in your README file. This is important to give proper credit and respect to the source of your inspiration.

Please update your README file and add a link to Cindy Shin’s designs. You can write something like this:
`This template was inspired by [Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
`

This will make your README file more informative and helpful for other people who want to check out her work. Thank you for your contribution and cooperation.